### PR TITLE
html-css-popup task from antonkottans added

### PR DESCRIPTION
**Pull request by AntonKottans**
### HTML & CSS Popup

[Demo](https://antonkottans.github.io/hooli-style-popup/)
[Code base](https://github.com/AntonKottans/hooli-style-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by lenadgit — CHANGES_REQUESTED** [review]
Could you please add the padding to a top row? Icons sticks to the top
top-icons

File: submissions/AntonKottans/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
```
Comment (lenadgit) [inline submissions/AntonKottans/html-css-popup/index.html:4]: According to the HTML specification - all single tags should be without slashes "/". Could you please remove all these slashes?
 https://codebra.ru/en/lessons-html/dating/2/1

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Review by lenadgit — CHANGES_REQUESTED** [review]
According to the HTML specification - all single tags should be without slashes "/". Could you please remove all these slashes?
https://codebra.ru/en/lessons-html/dating/2/1

File: submissions/AntonKottans/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                        id="popup-checkbox"
+                    />
+						<label for="popup-checkbox" class="popup-checkbox-label">
+							<img class="nav-image" src="img/popup-button.png" alt="" />
+						</label>
+						<div class="popup-menu">
+							<a href="#" class="popup-item"
+								><img
+									class="popup-item-img"
+									alt=""
```
Comment (lenadgit) [inline submissions/AntonKottans/html-css-popup/index.html:32]: Each attribute `alt` should contains a text, it is necessary for [a11y](https://developer.mozilla.org/en-US/docs/Web/Accessibility) . Disabilities view the web-page by screen-reader and it helps them to understand the context of each picture.
Could you please add the appropriated text to each `alt`

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Review by lenadgit — CHANGES_REQUESTED** [review]
Could you please format CSS-file by auto-formatting? (to add the same alignments to all lines)

**Comment by lenadgit** [comment]
In general the functionality is good, especially the animation for appearing of popup is a good opinion as a UX\UI solution. It remains to work a little on syntax of HTML5

File: submissions/AntonKottans/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	list-style: none;
+}
+
+a,
+a:visited {
+	text-decoration: none;
+	color: #606060;
+}
+
+hr.popup-hr {
```
Comment (A-Ostrovnyy) [inline submissions/AntonKottans/html-css-popup/style.css:22]: Avoid writing too specific CSS selectors. Also, avoid styling by HTML tag or `#id` [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by A-Ostrovnyy** [comment]
@lenadgit your turn)

**Review by lenadgit — APPROVED** [review]

**Pull request merged**